pyproject.toml

```diff
@@ -15,12 +15,12 @@ classifiers = [
   "Topic :: Home Automation",
 ]
 dependencies = [
-  "aiohttp",
-  "aiorun",
-  "async-timeout",
-  "coloredlogs",
-  "dacite",
-  "orjson",
+  "aiohttp==3.9.5",
```

Comment:
We don't want to pin any of the client dependencies.

Reply:
Right, to avoid Core getting into conflicting imports. I'll try adding the same dependencies to the server but with constraints.